pkg/compiler/compiler.go

```diff
@@ -59,6 +59,13 @@ func (c *Compiler) CompileTests() error {
 }
 
 func (c *Compiler) compileMain(binaryName, path string) error {
+	// do not build if binary is already there
```

Comment:
Can you, instead of having a comment here, have some godoc which explains what happens here in one or two sentences?

Reply:
Done, PTAL.